docs(agw): Update troubleshooting agw unable to checkin

wallyrb opened the pull request:
Signed-off-by: Wally Rodriguez B <wallyrb@fb.com>

<!--
    Tag your PR title with the components that it touches.
    E.g. "[lte][agw] Changeset" or "[orc8r][docker] ..."
-->

## Summary

- Fixing namespace in kubectl command
- Change pod name in the kubectl get logs, since controller is not longer a pod in Orc8r 1.5+

## Test Plan

Tested locally

![image](https://user-images.githubusercontent.com/37117037/124021946-07dfc680-d9a1-11eb-8e6d-1cb1beeac5f3.png)

wallyrb committed:
[docs] Update troubleshooting agw unable to checkin v2

Signed-off-by: Wally Rodriguez B <wallyrb@fb.com>

karthiksubraveti commented on docs/readmes/howtos/troubleshooting/agw_unable_to_checkin.md:
```diff
@@ -67,12 +68,12 @@ rootca_cert: /var/opt/magma/tmp/certs/rootCA.pem
 
-
-9. From Orchestrator, get all pods and find pod orc8r-controller-*
+9. From Orchestrator, get all pods and verify attempts from AGW are reaching Orc8r in nginx and look for any bootstrapping erors in the bootstrapper.
```
Add an example

wallyrb replied:
@karthiksubraveti I Added more details and example in the command

karthiksubraveti commented on docs/docusaurus/versioned_docs/version-1.5.0/howtos/troubleshooting/agw_unable_to_checkin.md:
````diff
@@ -68,12 +69,12 @@ rootca_cert: /var/opt/magma/tmp/certs/rootCA.pem
 ```
-kubectl -n magma get pods
-kubectl -n magma logs -f <controller podname>
+kubectl -n orc8r get pods
````
Add an example

karthiksubraveti reviewed (approved):
Left a comment. Kindly fix and merge.

themarwhal commented:
@wallyrb is this `ready2merge`?

wallyrb commented:
> @wallyrb is this `ready2merge`?

Sorry! I missed this notification, I added the tag

electronjoe commented:
> > @wallyrb is this `ready2merge`?
> 
> Sorry! I missed this notification, I added the tag

@wallyrb so we have now turned up the semantic commit message requirement - which this PR title fails. See [proposal](https://github.com/magma/magma/issues/7105) for details on commit message format!

I think for this PR it would be `docs(AGW): Update troubleshooting agw unable to checkin`?

wallyrb commented:
> > > @wallyrb is this `ready2merge`?
> > 
> > 
> > Sorry! I missed this notification, I added the tag
> 
> @wallyrb so we have now turned up the semantic commit message requirement - which this PR title fails. See [proposal](https://github.com/magma/magma/issues/7105) for details on commit message format!
> 
> I think for this PR it would be `docs(AGW): Update troubleshooting agw unable to checkin`?

thanks for the information @electronjoe. I have updated the tittle and also rebase the PR.

electronjoe commented:
Thanks @wallyrb !
